apps/server/package.json

```diff
@@ -21,6 +21,7 @@
     "ontime-utils": "workspace:*",
     "steno": "^3.1.0",
     "ts-essentials": "^9.4.1",
+    "valid-filename": "^4.0.0",
```

Comment:
My initial suggestion was to use filenamify instead. I am skeptic of the small usage of this library.

express has millions of weekly downloads, how is everyone doing file name validation?

dependabot recommends https://www.npmjs.com/package/sanitize-filename

Reply:
valid-filename and filenamify  are from the same people
 but [sanitize-filename](https://www.npmjs.com/package/sanitize-filename) looks good also, I will give that a try